build-tools/packages/build-tools/src/type-test-generator/main.ts

```diff
@@ -62,25 +63,42 @@ const currentFile = new Project({
 
 const previousTsConfigPath = `${previousBasePath}/tsconfig.json`;
 let previousFile: SourceFile;
-if (existsSync(previousTsConfigPath)) {
-	const project = new Project({
-		skipFileDependencyResolution: true,
-		tsConfigFilePath: previousTsConfigPath,
-	});
+const project = new Project({
+	skipFileDependencyResolution: true,
+	tsConfigFilePath: existsSync(previousTsConfigPath) ? previousTsConfigPath : undefined,
+});
+// Check for existence of alpha and add appropriate file
+if (existsSync(alphaFilePath)) {
+	project.addSourceFilesAtPaths(alphaFilePath);
```

Comment:
For my understanding: what does this line do?

Reply:
ts-morph uses Project instances. You add files to these Project instances and then all files in the project are available for analysis, manipulation etc